_ja/overviews/core/futures.md

```diff
@@ -454,12 +421,17 @@ Future は同じ `Throwable` とともに失敗する。
     }
     for (exc <- f.failed) println(exc)
 
-以下の例は画面に何も表示しない:
+以上のfor内包表記はこのように訳される
 
-    val f = Future {
+    f.failed.foreach( exc => println(exc))
+
+`f`が失敗したのために、クロージャは新しく成功した`Future[Throwable]`に
+登録される。 以下の例は画面に何も表示しない:
```

Comment:
原文: Because `f` is unsuccessful here, the closure is registered to
the `foreach` callback on a newly-successful `Future[Throwable]`.
The following example does not print anything to the screen:

```suggestion
`f` が失敗したため、クロージャは新規に成功値を持つ `Future[Throwable]` の
`foreach` コールバックに登録される。
以下の例は画面に何も表示しない:
```